__tests__/fixtures.ts

```diff
@@ -11,13 +11,15 @@ export const testEmployee: Employee = {
     firstName: "testi",
     lastName: "tekija",
     email: "test@test.test",
+    role: "USER",
```

Comment:
Even though uid is not a mandatory field you should include it in the tests, because hopefully we'll end up in a situation once the initial db syncing is done that we can make it a mandatory field